graphics/src/window/compositor.rs

```diff
@@ -38,6 +38,9 @@ pub trait Compositor: Sized {
         height: u32,
     );
 
+    /// Returns [`GraphicsInformation`] used by this [`Compositor`].
+    fn get_information(&self) -> GraphicsInformation;
```

Comment:
Renamed in https://github.com/iced-rs/iced/pull/1314/commits/005e516b5e1e8bb22f2da8524ffe4529f3b60ba1.

Since retrieving this information is done through a dependency (that uses system calls), I decided to stick with `fetch_`, although this is done synchronously.